### PR TITLE
Release 0.12.6

### PR DESCRIPTION
**Pull request by khoroshevskyi**
Updated windows support

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -54,6 +54,10 @@ geofetch -i GSE95654 --just-metadata
 geofetch -i GSE95654 --processed --just-metadata
 ```
 
+
+&#8291;**Note:** We ensure that GEOfetch is compatible with Unix, Linux, and Mac OS X. 
+However, due to dependencies, some features of GEOfetch may not be available on Windows.
```
Comment (nleroy917) [inline docs/README.md:59]: Can we just say which specific dependencies are not supported? I think this is fine either way.

File: geofetch/geofetch.py [hunk 2]
```diff
@@ -372,10 +372,17 @@ def fetch_all(self, input: str, name: str = None) -> Union[NoReturn, peppy.Proje
         # check to make sure prefetch is callable
         if not self.just_metadata and not self.processed:
             if not is_command_callable("prefetch"):
-                raise SystemExit(
-                    "To download raw data You must first install the sratoolkit, with prefetch in your PATH."
-                    " Installation instruction: http://geofetch.databio.org/en/latest/install/"
-                )
+                if os.name == "nt":
```
Comment (nleroy917) [inline geofetch/geofetch.py:375]: So if its windows, we don't prefetch, and just warn them instead of exiting?

File: geofetch/geofetch.py [hunk 3]
```diff
@@ -1789,15 +1796,22 @@ def _download_processed_file(self, file_url: str, data_folder: str) -> bool:
...
-                # so we should sleep a bit to reduce frequency
-                sleeptime = (ntry + 1) ** 3
-                _LOGGER.info(f"Sleeping for {sleeptime} seconds")
-                time.sleep(sleeptime)
-                ntry += 1
-                if ntry > 4:
-                    raise e
+                if os.name == "nt":
+                    _LOGGER.error(f"{e}")
+                    raise OSError(
```
Comment (nleroy917) [inline geofetch/geofetch.py:1801]: Should we raise an error here? I feel like we can just check for the `wget` in the beginning and handle it from there, no?

**Review by nleroy917 — CHANGES_REQUESTED** [review]
I feel like maybe there is a simpler way handle the `wget` not existing, right? instead of checking the OS? Not sure - if you've tested then its probably fine.

File: docs/README.md (hunk 1 above)
Comment (khoroshevskyi) [reply docs/README.md:59]: It's hard to say, everything should work fine, if you have all dependencies installed.
There is a problem with installing dependencies and checking if they exist.
If you want to spend some time on investigation this issue, you are welcome

File: geofetch/geofetch.py (hunk 2 above)
Comment (khoroshevskyi) [reply geofetch/geofetch.py:375]: yes, because I don't know how to check if prefetch exist in windows

File: geofetch/geofetch.py (hunk 3 above)
Comment (khoroshevskyi) [reply geofetch/geofetch.py:1801]: Same, here..

File: geofetch/geofetch.py (hunk 2 above)
Comment (nleroy917) [reply geofetch/geofetch.py:375]: Could we use `where`? Its available on windows: https://stackoverflow.com/questions/304319/is-there-an-equivalent-of-which-on-the-windows-command-line

File: geofetch/geofetch.py (hunk 3 above)
Comment (pedro-w) [reply geofetch/geofetch.py:1801]: Could you try to execute `wget --version` at the start and mark it unavailable if it raises an error?
In Powershell `wget` is aliased to `Invoke-WebRequest` so if you type `wget`, something will happen but it's not really wget (different command line args etc.) I don't know if Python will cause Powershell commands to be executed or not, but if it does, that could also cause confusion.

Comment (nleroy917) [reply geofetch/geofetch.py:1801]: Do you know if any of this is a problem when using WSL? I thought WSL leveled the playing field when it came to posix compatibility with windows. Unfortunately I don't have immediate access to a Windows machine to test some of these things.

But I kind of agree with @pedro-w in that it should be pretty easy to do an initial check at the beginning to see if `wget` is available at all and then raise errors/move forward from there

Comment (pedro-w) [reply geofetch/geofetch.py:1801]: It does appear to work for me under WSL but I only ran one command and saw that it started downloading files ([comment](https://github.com/pepkit/geofetch/issues/130#issuecomment-1888689387))
... but WSL could be one of a number of distros and might not have wget installed (same as Linux, actually)

**Pull request merged**